lib/wallaby/session.ex

```diff
@@ -124,6 +124,22 @@ defmodule Wallaby.Session do
   end
 
   @doc """
+  Gets the current url of the session.
+  """
+  @spec get_current_url(t) :: String.t
+  def get_current_url(session) do
+     Driver.current_url(session)
+  end
+
+  @doc """
+  Gets the current url of the session.
+  """
+  @spec get_current_url(t) :: String.t
```

Comment:
Can you update this to the correct spect? I think it should be `get_current_path`. Looks like the `@doc` needs to be updated as well.